res/translations/strings.en.po

```diff
@@ -160,7 +160,7 @@ msgstr "rusty swords"
 
 msgctxt "iteminfo"
 msgid "magicbag"
-msgstr "This bag encloses a dimensional fold, which can store up to 200 stones of weight without any extra burden on the bearer. Large items such as horses or carts cannot be placed inside."
+msgstr "This bag encloses a dimensional fold, which can store up to 200 stones of weight without any extra burden on the bearer. Large items such as stones, horses or carts cannot be placed inside. It is also not possible to transport one magic bag in another. The bag itself weighs 1 stones of weight"
```

Comment:
Habe ich irgendwo schonmal GE als stone (14lb) übersetzt? Wenn ja, ist das verwirrend. Außerhalb von England kennt die Gewichtseinheit niemand (insbesondere nicht in USA), und es kann Verwechslung mit Steienn auftreten (die in Eressea mehr als 14 Pfund wiegen).

Reply:
Die "200 stones of weight" waren schon drin, daher habe ich mich daran orientiert. Wie wird das denn sonst abgekürzt im deutschen steht da ja schlich 1GE